src/main/java/com/minecolonies/coremod/util/WorkerUtil.java

```diff
@@ -0,0 +1,186 @@
+package com.minecolonies.coremod.util;
+
+import com.minecolonies.api.util.EntityUtils;
+import com.minecolonies.api.util.Utils;
+import com.minecolonies.coremod.entity.EntityCitizen;
+import com.minecolonies.coremod.entity.pathfinding.PathResult;
+import net.minecraft.init.Blocks;
+import net.minecraft.pathfinding.PathPoint;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Vec3i;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Utility methods for BlockPos.
+ */
+public final class WorkerUtil
```

Comment:
Nice that this is finally split off from Entity utils.

However once enough abstraction has been done. This class should be moved into the API as it contains some usefull stuff (like Setting the SpawnPoint etc)